deploy/Jenkinsfile

```diff
@@ -344,9 +344,9 @@ pipeline {
         failure {
             script {
                 if (!env.ghprbPullId) {
-                    // mail to: 'bbp-deployment@groupes.epfl.ch',
-                    //      subject: "Failed Deployment Build: ${currentBuild.fullDisplayName}",
-                    //      body: "See ${env.BUILD_URL}"
+                    mail to: 'bbp-deployment@groupes.epfl.ch',
+                         subject: "Failed Deployment Build: ${currentBuild.fullDisplayName}",
+                         body: "See ${env.BUILD_URL}"
```

Comment:
this is for nightly CI plan? (or every build? I assume not)

Reply:
Every *failed* build. Better to be notified about those. And there is such a thing as email filters ;)

Reply:
@matz-e: I personally feel this will be too many emails for most of the people on the mailing list and hence my hesitation.

Reply:
@pramodk we've had silent failures, which is **very bad**. By large, the deployment succeeds, and failures should be resolved swiftly to avoid missing archive modules etc.

Also, there is a grand total of five (5) people on that mailing list [to be expanded]! Our OWA also allows to add server-side filters to make sorting out these emails trivial.

Reply:
> Also, there is a grand total of five (5) people on

thats what I considered when I said "most of the people" 😄 Anyway, go ahead with the merge and lets see if we should change this.